bridge/client/app/_models/project.ts

```diff
@@ -57,30 +58,6 @@ export class Project {
       ?.findTrace(trace => stage ? trace.isDeployment() == stage.stageName : !!trace.isDeployment());
   }
 
-  getLatestSuccessfulArtifact(service: Service, stage?: Stage): Trace {
-    let currentService = this.getService(service.serviceName);
-
-    if(currentService.roots)
-      return currentService.roots
-        .filter(root => (root.isEvaluation() || root.isDeployment()) && (!stage || root.isFaulty() != stage.stageName || root.isDeployment() && root.getDeploymentDetails(stage.stageName)?.isDirectDeployment()))
-        .reduce((traces: Trace[], root) => [...traces, ...root.traces], [])
-        .find(trace => stage ? trace.isDeployment() == stage.stageName || trace.isEvaluation() == stage.stageName : !!trace.isDeployment() || !!trace.isEvaluation());
-    else
-      return null;
-  }
-
-  getLatestArtifact(service: Service, stage?: Stage): Trace {
-    let currentService = this.getService(service.serviceName);
-
-    if(currentService.roots)
-      return currentService.roots
-        .filter(root => root.isEvaluation() || root.isDeployment())
-        .reduce((traces: Trace[], root) => [...traces, ...root.traces], [])
-        .find(trace => stage ? trace.isDeployment() == stage.stageName || trace.isEvaluation() == stage.stageName : !!trace.isDeployment() || !!trace.isEvaluation());
```

Comment:
Did you remove the part of considering an Evaluation to determine whether to show the message or not?